server/app.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const cors = require('cors');
+const db = require('../db/index.js');
```

Comment:
This db is the one that is connected to localhost:27017/syllabuses right? And it is used in your CRUD endpoints below yea?

Reply:
This `db` is the production version, yes, kind of - only difference is that I'm not using it's `connect()` function (which would connect it to prod database), so it's still connecting to the test db, not the production `/syllabuses` database. In this case, `db.rest` contains all the CRUD methods, so `db.rest.get()`, `db.rest.create()`, `db.rest.delete()`, etc.